builtin/credential/cert/backend_test.go

```diff
@@ -1652,6 +1691,7 @@ func testAccStepCert(
 			"allowed_uri_sans":             testData.uris,
 			"allowed_organizational_units": testData.organizational_units,
 			"required_extensions":          testData.ext,
+			"metadata_extensions":          testData.metadata_ext,
```

Comment:
Would it make sense to rename this to something like `allowed_metadata_extensions` ? I think having the field name reflect the purpose, that extensions with these oids are allowed to be added into identity metadata, like allowed_uri_sans / allowed_organizational_units, might be clearer ?

Reply:
I wanted to initially avoid `allowed_metadata_extensions` as the other `allowed_*` keys restrict access to the authentication endpoint for certificates lacking those properties. But I follow you, and it indeed makes sense to put `allowed_metadata_extensions`, as it is a filter on the oids that we'll accept to set as metadata.